### PR TITLE
Chore/make faq responsive

### PR DESCRIPTION
**Pull request by OlhaPo**
The following changes, which were not in accordance with the design, were made and require further discussion with frontend team:

1. To prevent images from appearing too small on screens between 1025px and 1240px, set the flex container's layout to flex-direction: column within this range.
Screenshot_9

2. Add an additional breakpoint to optimize image width transitions and prevent it from becoming too narrow or wide on specific screen sizes. 

3. To prevent excessive gaps between columns on larger desktops, apply justify-content: space-evenly starting at the $large-desktop breakpoint

### CODE REVIEW

File: .prettierrc [hunk 1]
```diff
@@ -3,8 +3,7 @@
   "bracketSpacing": true,
   "endOfLine": "lf",
   "jsonRecursiveSort": true,
-  "jsxSingleQuote": false,
-  "plugins": ["prettier-plugin-sort-json"],
```
Comment (biryukov-m) [inline .prettierrc:7]: turn it on

File: public/img/faq-section.webp
Comment (biryukov-m) [inline public/img/faq-section.webp:1]: Re-export this from the 1680p design with parameters .jpeg x2. The size will be almost half.
Screenshot 2024-12-12 at 14 04 36

Comment (biryukov-m) [reply public/img/faq-section.webp:1]: (saves us space when we will upload this to hosting)

File: src/features/home/sections/QASection/hooks/useQaData.ts [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  const imageAlt = translations('imageAlt');
+  const imgAuthor = translations('imgAuthor');
+  const messages = useMessages() as QaMessages;
+  const keys = Object.keys(messages?.homepage?.qASection?.questions ?? {});
+  const questions: IQuestion[] = keys.map((question) => ({
+    questionNumber: translations(`questions.${question}.questionNumber`),
+    title: translations(`questions.${question}.title`),
+    answer: translations(`questions.${question}.answer`),
+  }));
+
```
Comment (biryukov-m) [inline src/features/home/sections/QASection/hooks/useQaData.ts:31]: sort questions here.
This code can help

```suggestion
  // Sort questions by questionNumber in ascending order
  questions.sort((first, second) => {
    // eslint-disable-next-line require-unicode-regexp
    const aNumber = parseInt(first.questionNumber.replace(/\D/g, ''), 10);
    // eslint-disable-next-line require-unicode-regexp
    const bNumber = parseInt(second.questionNumber.replace(/\D/g, ''), 10);
    return aNumber - bNumber;
  });
```

**Review by biryukov-m — CHANGES_REQUESTED** [review]

**Review by biryukov-m — APPROVED** [review]

**Review by baza-trainee — APPROVED** [review]

**Pull request merged**